sdk/core/azure-core/inc/uuid.hpp

```diff
@@ -3,34 +3,35 @@
 
 #pragma once
 
+#include <memory>
```

Comment:
For std::memcpy, it is surprisingly `<cstring>` that you need to include: http://www.cplusplus.com/reference/cstring/memcpy/